qiime2/core/testing/plugin.py

```diff
@@ -100,33 +100,76 @@
     SingleIntFormat, RedundantSingleIntDirectoryFormat,
     citations=[citations['mayer2012walking']])
 
-dummy_plugin.register_semantic_type_to_format(
+
+# Create IntSequence1 import example usage example
+def is1_use(use):
+    def factory():
+        from qiime2.core.testing.format import IntSequenceFormat
+        from qiime2.plugin.util import transform
+        ff = transform([1, 2, 3], to_type=IntSequenceFormat)
```

Comment:
This is probably the best use of `transform` within a `plugin` context I've seen. I always forget this function exists.